src/index.js

```diff
@@ -49,7 +48,6 @@ const showList = () => {
 };
 
 // Delete Button to remove task
```

Comment:
- Use modules for the functionalities in this file. This will make your code easy to manage and test as it will be in a different file.

Reply:
Thank you will amend it accordingly.